src/pages/merge/bro/index.tsx

```diff
@@ -15,6 +17,16 @@ const BroMergePage = () => {
   const { data: broContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.broEater, CONTRACT_ADDRESSES.wewe);
   const { data: bbroContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.bbroEater, CONTRACT_ADDRESSES.wewe);
 
+  const getDaysRemaining = () => {
+    const today = new Date();
+    const startDate = new Date("2024-10-07 16:30:00");
```

Comment:
Date object in JS is relative to user's locale timezone, so countdown will be different in different regions. To fix it, we can pass timestamp.